packages/augur-tools/src/libs/ganache.ts

```diff
@@ -2,23 +2,36 @@ import * as ganache from "ganache-core";
 import { ethers } from "ethers";
 import memdown from "memdown";
 import { MemDown } from "memdown";
+import { Contracts as compilerOutput } from "@augurproject/artifacts";
 
 import { Account } from "../constants";
+import crypto from "crypto";
+import { EthersProvider } from "@augurproject/ethersjs-provider/build";
```

Comment:
are we standardizing to use top level references

Reply:
good catch!